packages/eds-core-react/src/components/SideSheet/SideSheet.docs.mdx

```diff
@@ -0,0 +1,56 @@
+import { Links, PropsTable, Story } from './../../../.storybook/components'
+import { SideSheet } from '.'
+
+# SideSheet
+
+A toggleable area that is anchored to the edge of the viewport with supplementary content.
+
+<Links 
+    figmaUrl="https://www.figma.com/file/0bGXR2sCwMVSDNyyzu5BXrO5/UI%E2%80%94User-Interface?node-id=17790%3A0"
+    documentationUrl="https://eds.equinor.com/0b0c666ab/p/17d96d-side-sheet/b/282ac1"
+    sourceUrl="https://github.com/equinor/design-system/blob/develop/packages/eds-core-react/src/components/SideSheet/SideSheet.tsx"
+    npmUrl="https://www.npmjs.com/package/@equinor/eds-core-react"
+/>
+
+<Story id="surfaces-sidesheet--introduction" />
+<PropsTable />
+
+## Usage
+
+`SideSheet` is surfaces containing supplementary content.
+
+<ul>
+    <li>Side sheets contain content that supplements the screen’s primary UI region.</li>
+    <li>Side sheets display a wide variety of content and layouts.</li>
+    <li>Side sheets can be visible or hidden based on screen size or user need, and they often become bottom sheets on mobile.</li>
+</ul>
+
+```tsx
+import { SideSheet } from '@equinor/eds-core-react'
+
+<SideSheet>
+  <Child>Children</Child>
```

Comment:
`Child` is not a component we have in our libraries so this example might be confusing. Maybe just replace it with a comment something like; `// your content here`